app/views/declaration/destinationCountries/country_of_routing.scala.html

```diff
@@ -26,6 +26,8 @@
 
 @(mode: Mode, form: Form[String], page: CountryPage)(implicit request: JourneyRequest[AnyContent], messages: Messages)
 
+@tariffBody = { @Html(messages("declaration.routingQuestion.help.bodyText", components.details_content_link())) }
```

Comment:
would you like to inline this varaible ?

Reply:
done